exercises/exercise3.js

```diff
@@ -1,12 +1,444 @@
+      } else {
+        return "You don't spell so good do you? Please try again.";
+      }
+    },
+    isListening: true,
+  },
+  {
+    label: "Start Game",
+    prompt: "Yes I am",
+    handler: function (params) {
+      return "Great! Now that we've hopefully got rid of those Florida people, let's start playing! \n This is a game to test your Geography skills. \n It's very simple, as long as you keep answering correctly, you keep progressing through the different levels. \n There are 16 levels and each level gets harder. \n To answer a question simply put in the question number followed by the answer, like this: Q1: Answer \n To start the game, answer this question: ________ is the study of places and the relationships between people and their environments.";
+    },
+    isListening: true,
+  },
+  {
+    label: "Question 1",
+    prompt: "Q1: #{answer1}",
+    handler: function (params, state) {
+      state.score = 0;
+      const answer1 = params.answer1;
+      if (answer1.toLowerCase() === "geography") {
+        return (
+          "Nice job " +
+          state.firstName +
+          ", " +
+          answer1 +
+          " is the correct answer. Well done! Your score is " + (state.score += 10) + "." + "\nNext question: What is the second biggest country in the world?"
+        );
+      } else {
+        return "Sorry, that's wrong. A person with your IQ should have a low voice too!";
+      }
+    },
+    isListening: true,
+  },
+  {
+    label: "Question 2",
+    prompt: "Q2: #{answer2}",
+    handler: function (params, state) {
+      const answer2 = params.answer2;
+      if (answer2.toLowerCase() === "canada") {
+        return (
+          "Nice job " +
+          state.firstName +
+          ", " +
+          `${
+            answer2.charAt(0).toUpperCase() + answer2.slice(1).toLowerCase()
+          }` +
+          " is the correct answer. Well done! Your score is " + (state.score += 10) + "." + "\nNext question: What is the longest river in Africa?"
+        );
+      } else {
+        return "Sorry, that's wrong. All that you are you owe to your parents. Why don't you send them a penny and square the account?";
+      }
+    },
+    isListening: true,
+  },
+  {
+    label: "Question 3",
+    prompt: "Q3: #{answer3}",
+    handler: function (params, state) {
+      const answer3 = params.answer3;
+      if (answer3.toLowerCase() === "nile") {
+        return (
+          "Nice job " +
+          state.firstName +
+          ", " +
+          "the " +
+          `${
+            answer3.charAt(0).toUpperCase() + answer3.slice(1).toLowerCase()
+          }` +
+          " is the correct answer. Well done! Your score is " + (state.score += 10) + "." + "\nNext question: Mumbai, Chennai, and Kolkata are metropolitan cities in which country?"
+        );
+      } else {
+        return "Sorry, that's wrong. Any similarity between you and a human is purely coincidental!";
```

Comment:
...but what if I typed 'The Nile'? Truly this function is well done, I just like to put 'the' in front of proper names so I got it wrong. Maybe a future change, to account for a 'the'?